controllers/storagecluster/readiness.go

```diff
@@ -0,0 +1,25 @@
+package storagecluster
+
+import (
+	"errors"
+	"net/http"
+
+	"sigs.k8s.io/controller-runtime/pkg/healthz"
+)
+
+var ready bool
```

Comment:
Rename this to something more specific, like `operatorReady`.

Reply:
Thanks for the feedback, I'll close this PR and reopen with the master branch, leaving out the CSV generation changes. I also changed the global variable name as posted here.